src/main/java/alfred/task/ToDo.java

```diff
@@ -0,0 +1,49 @@
+package alfred.task;
+
+import java.time.LocalDate;
+/**
+ * Encapsulates a <code>ToDo</code> which inherits from <code>Task</code>.
```

Comment:
Could I clarify what `<code>` header is for? On the pull request it just seems like there might be some formatting errors. Thanks!